app/schema/api/v3/models.rb

```diff
@@ -189,9 +189,11 @@ def facility
           config: {
             type: :object,
             properties: {
-              enable_diabetes_management: { type: :boolean } }
+              enable_diabetes_management: { type: :boolean },
+              enable_teleconsultation: { type: :boolean }
+            }
           },
-          required: %w[enable_diabetes_management]
+          required: %w[enable_diabetes_management enable_teleconsultation]
```

Comment:
If we expect this key to be present, can we add a `in: [true, false]` inclusion validation to the model?